src/commandline_content.ts

```diff
@@ -34,21 +35,17 @@ document.addEventListener("DOMContentLoaded", init)
 // This second call will fail in the most common case, but makes web-ext reloads effective.
 init()
 
-// The CSS in this file should probably go in static/
 export function show(){
-    // I don't understand how CSS works - but this ensures that the commandline is always at the bottom of the page.
-    cmdline_iframe.setAttribute("style", "position: fixed; bottom: 0; left: 0; z-index: 10000; width: 100%; height: 24px; border: 0; padding: 0; margin: 0;");
     const height = cmdline_iframe.contentWindow.document.body.offsetHeight + 'px'
-    cmdline_iframe.setAttribute("style", `position: fixed; bottom: 0; left: 0; z-index: 10000; width: 100%; height: ${height}; border: 0; padding: 0; margin: 0;`);
+    cmdline_iframe.setAttribute("style", `height: ${height};`);
```

Comment:
Just a little point on style - we aren't using semi-colons at the end of lines. I'm not sure why there was one in the original code.